go/worker/keymanager/init.go

```diff
@@ -70,6 +76,22 @@ func New(
 		return w, nil
 	}
 
+	for _, b64pk := range viper.GetStringSlice(CfgPrivatePeerPubKeys) {
+		pkBytes, err := base64.StdEncoding.DecodeString(b64pk)
+		if err != nil {
+			return nil, fmt.Errorf("oasis/keymanager: `%s` is not a base64-encoded public key (%w)", b64pk, err)
+		}
+		pk := signature.PublicKey{}
```

Comment:
This could just be `var pk signature.PublicKey`, and it would be slighyl more idiomatic.

It also occurs to me that `signature.NewPublicKey` would remove a lot of boiler-plate if it wasn't shit.